GraphRag/tools.py

```diff
@@ -0,0 +1,80 @@
+from llama_index.llms.ollama import Ollama
+from llama_index.core import SimpleDirectoryReader
+from llama_index.core.node_parser import CodeSplitter
+from llama_index.core import Document
+from llama_index.core import Settings
+
+
+
+
+def initialize_llm(base_url: str ="http://localhost:11434",model: str ="llama3",chunk_size: int = 512):
```

Comment:
Format your code using black tool. This review comment is meant for all the code files.

Reply:
Done! codes are formatted with black